java/lib/src/main/java/com/svix/Endpoint.java

```diff
@@ -128,4 +129,20 @@ public EndpointStats getStats(final String appId, final String endpointId) throw
 			throw Utils.wrapInternalApiException(e);
 		}
 	}
+
+	public void replay(final String appId, final String endpointId, final ReplayIn replayIn) throws ApiException {
+		try {
+			this.replay(appId, endpointId, replayIn, new PostOptions());
+		} catch (com.svix.internal.ApiException e) {
```

Comment:
Granted it's been a while since I've written Java, but the `try { } catch {}` here is redundant, no?

Reply:
Yep, also done.